src/data/adHocFilter.test.ts

```diff
@@ -152,6 +152,28 @@ describe('AdHocManager', () => {
     );
   });
 
+  it('apply ad hoc filter IN operator with string values', () => {
+    const ahm = new AdHocFilter();
+    ahm.setTargetTableFromQuery('SELECT * FROM foo');
+    const val = ahm.apply('SELECT stuff FROM foo WHERE col = test', [
+      { key: 'key', operator: 'IN', value: '(\'val1\', \'val2\')' },
```

Comment:
If we're adding a new `IN` operator, perhaps `value` could be more tailored to this use case.

For example, instead of having the user write out this complex escaped string `('val1', 'val2')`, it could instead just be comma separated `val1, val2` and then we format it in the code.

This would be similar to how we handle `IN` filters in the query builder:
- https://github.com/grafana/clickhouse-datasource/blob/7b50309aefba12ef9f2b980e2f9f255001b98db0/src/components/queryBuilder/FilterEditor.tsx#L93
- https://github.com/grafana/clickhouse-datasource/blob/7b50309aefba12ef9f2b980e2f9f255001b98db0/src/data/sqlGenerator.ts#L736-L738